.github/workflows/CI.yml

```diff
@@ -47,14 +52,33 @@ jobs:
         path-to-lcov: src/AzurePipelinesToGitHubActionsConverter.Tests/TestResults/coverage.info 
 
     #Pack the code into a NuGet package
-    - name: Pack with dotnet
-      run: dotnet pack src/AzurePipelinesToGitHubActionsConverter.Core/AzurePipelinesToGitHubActionsConverter.Core.csproj --configuration Release /p:PackageVersion=${{ steps.gitversion.outputs.nuGetVersionV2 }} /p:AssemblyVersion=${{ steps.gitversion.outputs.majorMinorPatch }} /p:Version=${{ steps.gitversion.outputs.majorMinorPatch }}
+    - name: dotnet pack
+      run: dotnet pack -c Release --nologo --no-build --include-symbols -p:Version='${{ steps.gitversion.outputs.semVer }}'
```

Comment:
this "--include-symbols" is the part that includes the debug symbols, right? It creates the .snupkg along with .nupkg?

Reply:
Yes, it increases the ease/surface-area with which package consumers can debug your library, see comparison;
![image](https://user-images.githubusercontent.com/16097639/119936946-40e3e000-bf8a-11eb-932a-6ef252862041.png)

Reply:
Clarifying question: If we simplified this entire PR - is this the *real* change you are trying to do here - and then a bunch of little incremental improvements to the project?

Reply:
yep

Reply:
Got it. Thanks for adding this functionality, I think it's useful and will help others. (plus I learned too :))